Sources/_StringProcessing/MatchingOptions.swift

```diff
@@ -63,6 +63,16 @@ extension MatchingOptions {
     stack.last!.contains(.reluctantByDefault)
   }
   
+  var defaultQuantificationKind: AST.Quantification.Kind {
+    if stack.last!.contains(.possessiveByDefault) {
```

Comment:
Is it the stack's last entry or any entry in the stack? Or are we merging options as we push?

Reply:
We merge options on every push, so all of these just look at the top of the stack.